src/platform/plugins/shared/share/public/components/export_integrations/export_integrations.tsx

```diff
@@ -328,7 +331,19 @@ function ExportMenuPopover({ intl }: ExportMenuProps) {
                 label={menuItem.config.label}
                 data-test-subj={`exportMenuItem-${menuItem.config.label}`}
                 isDisabled={menuItem.config.disabled}
-                onClick={() => openFlyout(menuItem)}
+                onClick={async () => {
```

Comment:
I also see the following behaviour: within the Lens app of a saved to the library viz, if I click the first time the `share` button, the `export` button is not triggerable the first time, I need to click on it two times to make it work